Demo/src/main/java/com/braintreepayments/demo/LocalPaymentFragment.java

```diff
@@ -35,18 +37,23 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         mIdealButton.setOnClickListener(this::launchIdeal);
 
         localPaymentClient = new LocalPaymentClient(requireContext(), super.getAuthStringArg());
-        localPaymentLauncher = new LocalPaymentLauncher(
-                localPaymentResult -> localPaymentClient.tokenize(requireContext(),
-                        localPaymentResult,
-                        this::handleLocalPaymentResult));
         return view;
     }
 
     @Override
     public void onResume() {
         super.onResume();
-        localPaymentLauncher.handleReturnToAppFromBrowser(requireContext(),
-                requireActivity().getIntent());
+        LocalPaymentPendingRequest.Started pendingRequest = getPendingRequest();
+        if (getPendingRequest() != null) {
+            LocalPaymentAuthResult paymentAuthResult = localPaymentLauncher.handleReturnToAppFromBrowser(pendingRequest, requireActivity().getIntent());
```

Comment:
nit - is there a coding style / line limit that the BT SDK adheres to?

Reply:
Rules are enforced for Kotlin files (max line length 120) but not for Java files currently 🤦‍♀️ - I just rely on the Android studio reformat file, but sometimes that adds unrelated formatting changes that distract from PRs. It's a work in progress. Updated this line though

Reply:
got it! it might not be a huge issue moving forward with more classes getting converted to kotlin.